pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "0.0.3.dev7"
+version = "0.0.3.dev8"
```

Comment:
A question for my edification :  Does `invoke version-bump` do this automatically?

Reply:
Yes